Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -181,7 +181,7 @@
 
 void RenderPassEncoder::setBlendConstant(const WGPUColor& color)
 {
-    UNUSED_PARAM(color);
+    [m_renderCommandEncoder setBlendColorRed:color.r green:color.g blue:color.b alpha:color.a];
```

Comment:
No bounds checking?

Also, this should really be a real test, not a demo.

Reply:
What type of bounds checking would you apply to colors? They can be any value positive or negative. We could make sure we don't pass NaNs or infs but I'm not sure if that is explicitly prohibited

Reply:
Oh wait WGPUColor is double whereas these are floats

Reply:
We could check for the values > FLT_MAX, < -FLT_MAX but would we abandon the operation or pass +/-inf? If we pass the inf values then there is no need to check